sdk/servicebus/azure-servicebus/samples/sync_samples/send_queue.py

```diff
@@ -14,32 +14,41 @@
 import os
 from azure.servicebus import ServiceBusClient, Message
 
+import logging
+logging.basicConfig(level=logging.INFO)
+
 CONNECTION_STR = os.environ['SERVICE_BUS_CONNECTION_STR']
 QUEUE_NAME = os.environ["SERVICE_BUS_QUEUE_NAME"]
 
 
 def send_single_message(sender):
-    message = Message("DATA" * 64)
-    sender.send_messages(message)
+    message = Message("Single Message")
+    sender.send(message)
```

Comment:
```suggestion
    sender.send_messages(message)
```
(throughout this file)

Reply:
updated!